js/all-the-ways-to-define-functions-in-js/all-the-ways-to-define-functions-in-js.md

```diff
@@ -0,0 +1,39 @@
+# All the ways to define functions in JavaScript
+
+## Introduction
+
+Function is a set of statements or block of code to perform a specific task and return a value.They are reusuable and make your program modular and efficient.
+
+Below are few of the ways by which you can define a function in JS.
+
+### Defining a Function
+
+#### 1. Function Statements / Function Decalaration / Function Definition
+
+This is the most common way to define a function in JS. The definition starts with the keyword **function**, followed by the **name** of the function where the naming convention is similar the variables - they can contain letters, digits, underscores and dollarsigns and are mostly written in **camelCase**. The name is followed by a set of parenthesis which is used to pass **arguments**. After this comes the curly brackets which contains the **body** of the function.
```

Comment:
This is the most common way to define a function in JS. The definition starts with the keyword **function**, followed by the **name** of the function where the naming convention is similar to the variables - they can contain letters, digits, underscores and dollar signs and are mostly written in **camelCase**. The name is followed by a set of parentheses which is used to pass **arguments**. After this comes the curly brackets which contain the **body** of the function.  
*Some minor grammatical fixes*